Dockerfile

```diff
@@ -46,9 +46,12 @@ set -eux
 # .cargo/config.toml
 cd /src
 
-# use the cache mount
-# (we will not be able to to write to e.g `/src/target` because it is bind-mounted)
-CARGO_TARGET_DIR=/artifacts cargo build --locked "--profile=${PROFILE}" --all
+# statically-link the C runtime for runtime performance
+RUSTFLAGS="-C target-feature=+crt-static" \
+    # use the cache mount
+    # (we will not be able to to write to e.g `/src/target` because it is bind-mounted)
+    CARGO_TARGET_DIR=/artifacts \
+    cargo build --locked "--profile=${PROFILE}" --target=x86_64-unknown-linux-gnu --all
```

Comment:
I don't think we should fix the `target` like this - we have users on `amd64` that this build needs to work for

Reply:
It's a good point, but yeah the target needs to be set explicitly for `crt-static`.

Maybe the target should be made an argument to the Dockerfile, instead?